features/themes/onms-default-theme/src/main/java/VAADIN/themes/opennms/styles.scss

```diff
@@ -200,3 +200,400 @@ $header-background-image: url("images/headerbg.png") !default;
   filter: alpha(opacity=25);
   */
 }
+
+/*  --------------------------------------------------
+    :: Surveillance View styles
+    -------------------------------------------------- */
+
+.surveillance-view .v-table-cell-content {
+  padding: 0px 0px 0px 0px;
+  background-color: #ffffff;
+}
+
+/*
+.v-table-cell-content-marked {
+  background: #57a7ed;
+  color: #fff;
+  padding: 0px 0px 0px 0px;
+}
+*/
+
+.v-caption-surveillance-view {
+  background-color: #4d9c29;
+  border-color: #4d9c29 !important;
+  border-bottom: 1px solid transparent;
+  padding: 6px;
+  color: white !important;
+  font-size: 12px !important;
+  font-weight: bold;
+
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  background-image: url(../../../../images/headingbg.png);
+  background-repeat: repeat-x;
+  background-position: center center;
+}
+
+.surveillance-view-fake-caption {
```

Comment:
This entry is used to create a similar header for the upper table with the dropdown 
box on the right. So, it's not the standard Vaadin table caption.